static.config.js

```diff
@@ -106,18 +110,20 @@ export default {
         component: 'src/js/pages/Departments',
         getData: async () => ({
           allDepartments: departmentQueries[langCode],
+          langCode: langCode,
         }),
-        children: departmentQueries[langCode].edges.map(department => ({
-          path: `${department.node.id}`,
+        children: departmentQueries[langCode].edges.map(({node:department}) => ({
+          path: `${department.id}`,
           component: 'src/js/pages/Department',
           getData: async () => ({
-            department: department.node,
+            department: department,
+            langCode: langCode,
           })
         }))
       },
       {
         path: '/search',
-        component: 'src/js/pages/Search',
+        component: 'src/js/pages/Search', //TODO: update 404 page to be conscious of all languages
```

Comment:
I think in this comment you meant `//TODO: update Search page to be conscious of all languages`

Which also makes me think about something for the first time. How do we support multi-language search. cc/ @benweatherman

Reply:
yep. and YIKES.